src/Components/Menu.tsx

```diff
@@ -1,13 +1,27 @@
 import React from "react";
-import { Button, Box, Typography, FormGroup, FormControlLabel, Switch } from '@material-ui/core';
+import { Button, Box, Typography, FormGroup, FormControlLabel, Switch, TextField, Menu, MenuItem } from '@material-ui/core';
 
 interface Inputs {
   refreshIntervalEnabled: boolean;
   setRefreshIntervalEnabled(x): void;
   menuClass: string;
+  setAnchorEl(x): void;
+  anchorEl: any;
+  refreshIntervalUnits: string;
+  setRefreshIntervalUnits(x): void;
+  refreshInterval: number;
+  setRefreshInterval(x): void;
+  setTimerRunning(x): void;
+  timerRunning: boolean;
+  setMenuError(x): void;
+  menuError: boolean;
 }
 
-export function Menu(Props: Inputs) {
+export function MenuView(Props: Inputs) {

```

Comment:
Parameter should be lowercase first letter.

Reply:
Are you talking about "Props"?

Reply:
Yes.